packages/react-sdk/src/wrappers/LivestreamPlayer/LivestreamPlayer.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+import { Call } from '@stream-io/video-client';
+import { useStreamVideoClient } from '@stream-io/video-react-bindings';
+import {
+  LivestreamLayout,
+  LivestreamLayoutProps,
+  StreamCall,
+} from '../../core';
+
+export type LivestreamPlayerProps = {
+  /**
+   * The call type. Usually `livestream`.
+   */
+  callType: string;
+  /**
+   * The call ID.
+   */
+  callId: string;
+  /**
+   * The props for the {@link LivestreamLayout} component.
+   */
+  layoutProps?: LivestreamLayoutProps;
+};
+
+export const LivestreamPlayer = (props: LivestreamPlayerProps) => {
+  const { callType, callId, layoutProps } = props;
+  const client = useStreamVideoClient();
+  const [call, setCall] = useState<Call>();
+  useEffect(() => {
+    if (!client) return;
+    const myCall = client.call(callType, callId);
+    setCall(myCall);
+    myCall.join().catch((e) => {
```

Comment:
Sorry for neat-picking but I believe try-catch would be better here rather than .catch, since this is the pattern we follow most of the places in the code.